example/example/autoextract.py

```diff
@@ -10,7 +10,8 @@
 from twisted.internet.threads import deferToThread
 
 from scrapy_po.page_input_providers import PageObjectInputProvider, provides
-from scrapy_po.webpage import Injectable, ItemPage
+from scrapy_po.webpage import ItemPage
+from scrapy_po.utils import DummyResponse
```

Comment:
Technically, DummyResponse could belong to page_inputs.py module, but I see why you put it to utils. No specific suggestions here :)